db/migrate/20220707172003_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, foreign_key: true
+  end
```

Comment:
- [ ] It should be the author, not the user. Kindly change it.
[Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_1.md#:~:text=Table%20and%20column%20names%20should%20match%20the%20ERD%20diagram.)
You can do it like in the picture.
![image](https://user-images.githubusercontent.com/38635158/177837184-ec62ca8a-3a06-4c56-b300-fa2820ebe958.png)